src/In.ProjectEKA.HipService/Link/Patient/LinkPatientController.cs

```diff
@@ -32,10 +32,7 @@ public LinkPatientController(ILink linkPatient)
                 request.Patient.CareContexts);
             var patientReferenceRequest = new PatientLinkReferenceRequest(request.TransactionId, patient);
             var (linkReferenceResponse, error) = await linkPatient.LinkPatients(patientReferenceRequest);
-            if (error != null)
-            {
-                return NotFound(error);
-            }
+            if (error != null) return NotFound(error);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4839081)